start.py

```diff
@@ -25,8 +25,10 @@ def main():
         print 'Could not activate virtual environment at "%s": %s.' % (JENKINS_ENV, str(ex))
         sys.exit(1)
 
-    # Download the Jenkins WAR file
+    from mozprocess.processhandler import ProcessHandler
     from mozdownload import DirectScraper
```

Comment:
>> @@ -25,8 +25,10 @@ def main():
>>          print 'Could not activate virtual environment at "%s": %s.' % (JENKINS_ENV, str(ex))
>>          sys.exit(1)
>>  
>> -    # Download the Jenkins WAR file
>> +    from mozprocess.processhandler import ProcessHandler
>>      from mozdownload import DirectScraper
> 
> I wonder if you have the sort algorithm ready to use in mind... !

Well, it's simply that mozd comes before mozp in the alphabet. :) Sorry
for being nitpicky here.